### DISCUSSION TITLE
Episode 204: 🇪🇸 Lambda World Live 🇪🇸

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#103] [Episode]
Episode Link: https://adspthepodcast.com/2024/10/18/Episode-204.html

Feel free to write comments or ask questions 🙂

**Comment by Brian-M-J** [comment]
Speaking of the "Can Programming be Liberated from the von Neumann Style" talk, there's a bunch of research now on computers that don't use the von Neumann architecture because of the "memory wall" that high performance applications are hitting. An example: [No-ISA is the Best ISA](https://youtu.be/G4fxdHozm5I).

Others try to defend and update it. One such paper: [Revising the Classic Computing Paradigm and Its Technological Implementations](https://www.mdpi.com/2227-9709/8/4/71).

**Reply by codereport** [reply]
Thanks for the links! Will definitely take a look at these : )
